### ISSUE TITLE
Grid - Aria PosInSet and SetSize Used on Wrong Role

### ISSUE THREAD

**Issue by tabrumle** [microsoft/fluentui#13532]
### Describe the issue:

The table element in grid is using set-size and pos-in-set aria attributes but does not have the supported role. Grid's role should be one of the supported roles specified by https://www.w3.org/TR/wai-aria-1.1/#aria-setsize guidelines

Would affect user experience if focus is put on the table. It is also causing issues with automation to detect accessibility issues.

![image](https://user-images.githubusercontent.com/42159057/84177061-3063ee00-aa37-11ea-867a-1103324a04a1.png)

**Comment by paulgildea** [comment]
@dzearing Looking through the documentation and codeowners file and there's no reference to a Grid component. However in the source code I found a Grid Component. Is Grid a helper component or is it recommended to be used on its own?

Either way - it should be implementing the correct ARIA pattern.

@tabrumle In what context are you using this component? Just so we understand the usage scenarios so that it has the correct behavior.

**Comment by tabrumle** [comment]
It is being used within a component to help organize buttons within a callout

**Comment by jdhuntington** [comment]
Thanks @tabrumle -- Are you willing to submit a PR to fix the issues you're seeing?

**Comment by tabrumle** [comment]
I can submit a PR with a fix. For the fix I plan on changing the role to "option".

**Comment by paulgildea** [comment]
@jdhuntington @dzearing - Can you help @tabrumle with a PR for the fix?

**Comment by paulgildea** [comment]
@jdhuntington @dzearing Thoughts on this folks?

**Comment by xugao** [comment]
@tabrumle - I don't think this is a bug. Here are some things you should know:
1. It's correct that by default the control has role be `grid`.
3. You can overwrite `role` by setting `role` prop (e.g. `role="option"`)on the `Grid`.
2. the aria props that you pointed out are optional, we shouldn't be adding these if it's causing you violations
```
  /**
   * Position this grid is in the parent set (index in a parent menu, for example)
   */
  ariaPosInSet?: number;

  /**
   * Size of the parent set (size of parent menu, for example)
   */
  ariaSetSize?: number;
```

Let me know if this helps
